dvc/command/get.py

```diff
@@ -40,10 +40,10 @@ def add_parser(subparsers, parent_parser):
         formatter_class=argparse.RawDescriptionHelpFormatter,
     )
     get_parser.add_argument(
-        "url", help="URL of Git repository with DVC project to download from."
+        "url", help="URL of Git repository to download from."
```

Comment:
```suggestion
        "url", help="Location of DVC project or Git repository to download from"
```
p.s. it can simply be a local file system path, even to a non-Git DVC project dir, I think.

Reply:
The fact this is called "url" might be confusing because of that.

Reply:
Yeah and also `path` since URL can be a path too... They're edge cases and explained in the docs so it should be OK.